src/routes/Logout.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
+import { Title } from '@patternfly/react-core';
+
+import './Logout.scss';
 
 const Logout = () => {
     return (
-        <section page-type="" className="pf-l-page__main-section pf-c-page__main-section">
-            Logout
+        <section className="pf-l-page__main-section pf-c-page__main-section ins-c-page__logout">
+            <Title size='3xl'> You have logged out of Cloud Services </Title>
+            <Title size='xl'><a onClick={ () => window.insights.chrome.auth.login() }>Log back in</a></Title>
```

Comment:
Can you add href as well since we are sending user away.

Reply:
Where do you want me to href to?

Reply:
To `window.insights.chrome.auth.login()`